src/Schema/Directives/MutationExecutorDirective.php

```diff
@@ -70,7 +70,10 @@ public function __invoke($parent, $args)
     }
 
     /**
+     * @template TModel of \Illuminate\Database\Eloquent\Model
```

Comment:
The generic only makes sense when used in multiple places. Think of it like a variable, only makes sense when used multiple times too. Should this be part of the return?